src/main/scala/jigg/util/JSONUtil.scala

```diff
@@ -35,27 +42,24 @@ object JSONUtil {
     def hasAttribution(x: Node): Boolean = !x.attributes.isEmpty
   }
 
-  def toJSON(x: Any): StringBuilder = x match {
-    case x: String if x.endsWith(".xml")    => toJSONFromNode(XML.loadFile(x))
-    case x: Elem      => toJSONFromNode(x)
-    case _            => new StringBuilder("")
-  }
+  def readJSON(file: String): String = toJSONFromNode(XML.loadFile(file))
```

Comment:
Sorry to complain again. Maybe `readJSON` here does not tell the meaning of the method correctly. Perhaps this method can be removed (and prohibit users to recover json from XML file directly in one method), which seems more safe?